pkg/crc/machine/machine.go

```diff
@@ -93,15 +93,19 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		result.Status = vmState.String()
 	} else {
-		logging.InfoF(" Starting stopped VM ...")
 		host, err := libMachineAPIClient.Load(machineConfig.Name)
 		s, err := host.Driver.GetState()
 		if err != nil {
 			logging.ErrorF("Error getting the state for host: %s", err)
 			result.Error = err.Error()
 		}
+		if s == state.Running {
```

Comment:
If we have this check early on in the start command, then we can prevent performing the preflight checks. I am for putting it there.. but since all the machine stuff is here added it here, would like to know what @gbraad @praveenkumar thinks!

Reply:
I don't know, this is like early feedback that machine already running ?

Reply:
Yes, i am for doing this check in the start command.. even before calling `machine.Start()` and starting the preflight checks..

Reply:
This is a different issue... valuable, but no need to address here